javascript/javascript1/week4/creditCardFormat.js

```diff
@@ -0,0 +1,20 @@
+function formatCreditCardNumber(number) {
+    if (typeof number == "string") {
+      console.log("Enter a valid card number");
+    } 
+    else {
+       number = number.toString();
```

Comment:
it is not recommended to use the same variable as a number and a string at the same time. Preferred way is 
`let myNumberString = number.toString();`